lira/Token.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Lira;
+	
+public sealed class Token
+{
+	readonly TokenKind Kind;
```

Comment:
For this and the rest, they should be declared as such:
`public Kind { get; } `

Reply:
why not public readonly?

Reply:
@cabouezzi Because it is more idiomatic.

Reply:
> @cabouezzi Because it is more idiomatic.

you calling me an idiot?